internal/testgen/gen_negative_positive.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"strings"
+	"text/template"
+
+	"github.com/Antonboom/testifylint/internal/checkers"
+)
+
+type NegativePostiveTestsGenerator struct{}
+
+func (NegativePostiveTestsGenerator) Checker() checkers.Checker {
+	return checkers.NewNegativePositive()
+}
+
+func (g NegativePostiveTestsGenerator) TemplateData() any {
+	var (
+		checker = g.Checker().Name()
+		report  = checker + ": use %s.%s"
+	)
+
+	return struct {
+		CheckerName       CheckerName
+		InvalidAssertions []Assertion
+		ValidAssertions   []Assertion
+		IgnoredAssertions []Assertion
+	}{
+		CheckerName: CheckerName(checker),
+		InvalidAssertions: []Assertion{
+			{Fn: "Greater", Argsf: "0, a", ReportMsgf: report, ProposedFn: "Positive", ProposedArgsf: "a"},
```

Comment:
please, support cases

```
True(a > 0)
True(0 > a)
False(a < 0)
False(0 < a)
```
(`compares` can be disabled and I allow some overlap in the linters)

+ similar ignored assertions

Reply:
Good catch. I didn't think about these ones.

Reply:
I started working on this, but then it leads to a very complex code in negative_positive.go

It has to replicate a lot of what `compares.go` does

I remember now why I initially stated to put negative-positive after compares.go

It's because compares would request to transform `True(t, a>0)` into `Greater(t, 0, a)`

From my perspective, and because you explained it well here, it should be the way to do.

https://github.com/Antonboom/testifylint?tab=readme-ov-file#chain-of-warnings

I mean compare is intended to detecte the True/False+boolean

https://github.com/Antonboom/testifylint?tab=readme-ov-file#compares

While negative-compare would detect

https://github.com/Antonboom/testifylint/blob/master/CONTRIBUTING.md#negative-positive

I don't think it's a good idea to mix everything.

Reply:
Please, provide an example of "very complex code". Compares is pretty simple and we need just of part of it.

How I wrote above 

> compares can be disabled and I allow some overlap in the linters

Need to find a balance. If implementation is really complex, then I can agree 👍 

"Chain of warning" is more about full independent linters. But in this case I feel some false negatives.

Reply:
I understand you disagree, but here is my point of view:
- compares is about catch True(a > b) and co to use Greater (and co)
- negative-positive is about detecting Greater(a, 0) to use Positive(a) and co

I think our visions differ, I dislike that, but it's OK.

Now, that is said, I implemented the feature that was requested and described here. 

https://github.com/Antonboom/testifylint/blob/7346760ab41bdd2fe72302620199d89725e0b8ed/CONTRIBUTING.md#negative-positive

We can argue and iterate on the need to add `True(a > 0)` (and co) detection to `negative-positive.go`
I have no problem in bringing a new feature such as this one in negative-positive.go, but I would prefer to do it in another PR, with a dedicated issue.

The reason is that bringing positive-negative now seems to me to be a better option for people using your tool.
So can you consider reviewing current code and merging it as-is 

For records, I'm more interested in adding other feature such as zero or error-contains than adding this edge case that is already reported by True(a >0), that will be reported to use Positive(a) on next run.

Reply:
Hi, @ccoVeille 

thank you for reply
I found some bugs in your PR and want to contribute and test it again on my own – to not waste your time more.

> I'm more interested in adding other feature

welcome ❤️ 

I returned to this PR soon and will describe changelog

Reply:
Come back to me when you can. I'm starting working again on "zero" checker

Reply:
> Please, provide an example of "very complex code". Compares is pretty simple and we need just of part of it.
> 
> How I wrote above 
> 
> > compares can be disabled and I allow some overlap in the linters
> 
> Need to find a balance. If implementation is really complex, then I can agree 👍 
> 
> "Chain of warning" is more about full independent linters. But in this case I feel some false negatives.

The code you added via 21776049f952ac584f9603b309ffa8f1edc1d0fe is great, but I would dare to say it's more complex than the code I wrote 😅

I will review the changes thanks